tests/unit/test_redshift.py

```diff
@@ -593,20 +593,20 @@ def test_spectrum_decimal_cast(
     # Athena
     df2 = wr.athena.read_sql_table(table=glue_table, database=glue_database)
     assert df2.shape == (2, 5)
-    df2 = df2.drop(df2[df2.c0 == 2].index)
```

Comment:
The two rows were both indexed at `0`. So even though this fetched the index of the correct rows, it ended up deleting both rows. I changed it to simple Pandas filter